contracts/sol6/KyberStorage.sol

```diff
@@ -15,7 +14,10 @@ import "./utils/Utils5.sol";
  *   - Stores feeAccounted data for reserve types
  *   - Record contract changes for network, matchingEngine, feeHandler, reserves, network proxies and kyberDAO
  */
-contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
+contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers {
+    IERC20 internal constant ETH_TOKEN_ADDRESS = IERC20(
+        0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE
```

Comment:
didn't follow
Why can't we use utils?

Reply:
Should we use it just to use ETH_TOKEN_ADDRESS?